### PR TITLE
typst: new, 0.0.0+git2023032

### PR DESCRIPTION
**Pull request by Nyovelt**
Trying to introduce typst, A LaTeX killer.

Topic Description
-----------------

No
Package(s) Affected
-------------------

No
Security Update?
----------------

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [ ] AArch64 `arm64`
    

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [ ] AMD64 `amd64`   
- [ ] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

### CODE REVIEW

File: app-doc/typst/spec [hunk 1]
```diff
@@ -0,0 +1,4 @@
+VER=0.0.0+git2023032
```
Comment (MingcongBai) [inline app-doc/typst/spec]: Upstream does have a tag - https://github.com/typst/typst/tags.

File: app-doc/typst/spec [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+REL=1
+SRCS="git::commit=a253b47e7c6f5e9500a40ac73f949a431f62289d::https://github.com/typst/typst.git"
+CHKSUMS="SKIP"
```
Comment (MingcongBai) [inline app-doc/typst/spec:6]: CHKUPATE missing.

File: app-doc/typst/autobuild/defines [hunk 3]
```diff
@@ -0,0 +1,8 @@
+PKGNAME=typst
+PKGSEC=doc
+PKGDES="A new markup-based typsetting system that is designed to be as powerful as LaTeX while being much easier to learn and use."
```
Comment (MingcongBai) [inline app-doc/typst/autobuild/defines]: This is too verbose for a short description and it forces a lot of subjective descriptions. Try this...

A markup-based typsetting system

This should be good enough.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: app-doc/typst/spec (hunk 1 above)
Comment (Nyovelt) [reply app-doc/typst/spec]: The thing is, based on this commit, https://github.com/typst/typst/commits/main, the release cannot be compiled unless patched to the latest commit.

**Review by MingcongBai — DISMISSED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request closed without merging**